src/main/app/src/app/zaken/besluit-edit/besluit-edit.component.ts

```diff
@@ -108,13 +119,49 @@ export class BesluitEditComponent implements OnInit, OnDestroy {
       .validators(Validators.required)
       .build();
 
+    this.divider = new DividerFormFieldBuilder().id("divider").build();
```

Comment:
Why do we need to add `this` to all these lines? Is there a conflict in names? Should we use different names somewhere?

Reply:
This my signature; since there is a block added and removed dynamically, I find adding this divider make that mor clear; it being a separate block.